canon/CHANGELOG.md

```diff
@@ -0,0 +1,84 @@
+# 0.6.0
+
+## Changed
+
+Deprecate the usage of the `Store` trait.
+
+the trait `Canon<S>` becomes `Canon`.
+
+`Canon::<S>::write(&type, &mut Sink<S>)?;` becomes `type.encode(&mut sink);`
+
+Writes are assumed to always succeed, analgous to how a push to a `Vec` in rust can only panic with out of memory exceptions, but is assumed to be failsafe.
+
+The names encode/decode is choosen not to confuse them with `std::io::{Read, Write}`.
+
+# 0.5.3 a9b53e4e0ebe96fa7bd90ce2ce03875d5e21124a
+
+# Removed
+
+Page size limitation removed on encoding/decoding
+
+## Changes
+page size to 32k
+
+# 0.5.2 81eadfa974f17362c223af11fd1d4765a14a1b51
+
+## Changes
+page size increase
+
+# 0.5.1 21e85f6ec78e96cd438d078b47b9739d5917759f
+
+REVERTED
```

Comment:
We should actually  add what the `0.5.1` was, any change that are reverted from there will be mentioned in `0.5.2`.